src/common/components/play/multiplayer/MultiplayerLobby.js

```diff
@@ -133,7 +133,8 @@ export default class MultiplayerLobby extends React.Component {
             myClientId={clientId}
             playersOnline={playersOnline}
             userDataByClientId={userDataByClientId}
-            onConnect={onConnect} />
+            onConnect={onConnect}
+            history={history} />
```

Comment:
Is `history` being used in `LobbyStatus`?

Reply:
Hmm not anymore I'll fix that